assets/js/blocks/featured-category/block.js

```diff
@@ -167,22 +167,26 @@ class FeaturedCategory extends Component {
 						},
 					] }
 				>
-					<RangeControl
-						label={ __( 'Background Opacity', 'woo-gutenberg-products-block' ) }
-						value={ attributes.dimRatio }
-						onChange={ ( ratio ) => setAttributes( { dimRatio: ratio } ) }
-						min={ 0 }
-						max={ 100 }
-						step={ 10 }
-					/>
-					{ !! FocalPointPicker && !! url &&
-						<FocalPointPicker
-							label={ __( 'Focal Point Picker' ) }
-							url={ url }
-							value={ focalPoint }
-							onChange={ ( value ) => setAttributes( { focalPoint: value } ) }
-						/>
-					}
+					{ !! url && (
+						<Fragment>
+							<RangeControl
+								label={ __( 'Background Opacity', 'woo-gutenberg-products-block' ) }
+								value={ attributes.dimRatio }
+								onChange={ ( ratio ) => setAttributes( { dimRatio: ratio } ) }
+								min={ 0 }
+								max={ 100 }
+								step={ 10 }
+							/>
+							{ !! FocalPointPicker &&
```

Comment:
I'm assuming that conditional exists because `FocalPointPicker` is introduced in a version of WordPress greater than the minimum Woo supports (so it may not be present in some installs)?  I know this is pre-existing, but I wonder if we should put a comment in here regarding what version of WP this is required (I'm guessing based on it's history it is WP 5.2) for future maintainers to know:

a. Why the condition exists.
b. When it can be removed.

Reply:
Done in 2804f71dbd3b043867da72864f008ee9592763d4.